nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-core/src/main/java/org/nuxeo/ecm/platform/tag/FacetedTagService.java

```diff
@@ -68,6 +66,11 @@ public boolean hasFeature(Feature feature) {
     }
 
     @Override
+    public boolean supportTag(CoreSession session, String docId) {
+        return session.getDocument(new IdRef(docId)).hasFacet(TAG_FACET);
```

Comment:
This is rather inefficient but we don't have a better way with the current public APIs...